docs/strat-lib/guides/approvals.md

```diff
@@ -1,201 +1,106 @@
 
-When taking an offer using a contract without a router, you only need to approve that the taker contract can transfer the outbound token from the takers reserve. If the taker didn't set any reserve, then the reserve will be the takers own address. E.g., if you want to take an DAI-USDC offer where you are buying DAI for USDC, then you need to approve that the taker contract can take USDC from the takers reserve.
+Strat Lib routers provide two methods to assist with their approvals:
 
-```mermaid
-sequenceDiagram
+* [`activate`](../technical-references/router.md#router-activation) performs all router-centric approvals necessary to route liquidity for a given token.
+* [`checklist`](../technical-references/router.md#router-checklist) verifies that router has necessary approvals to route liquidity for a given token and a given reserve (see [below](#when-the-reserve-is-not-directly-on-the-maker-or-the-router) for more on reserve approvals).
 
-    participant Taker as Takers reserve
-    participant TC as Taker Contract
-    rect grey
-    Taker ->> TC : Outbound
-    end
-    TC ->> Mangrove : 
-    Mangrove ->> Maker: 
-    Maker ->> Mangrove: 
-    Mangrove ->> TC: 
-    TC ->> Taker: 
-```
+Please refer to the section on [Routers](../technical-references/router.md) for more details, and refer to the API Reference for [AbstractRouter](../technical-references/code/strategies/routers/AbstractRouter.md) - the base that Strat Lib routers are implemented on top of.
 
-### Admin of a Maker contract with a router
 
-When you are an admin of a contract that can make offers, then you need to make sure that the contract itself has made the correct approvals. In this case we are using a %%router|router%%, so we not only need to approve Mangrove to make transfer from the contracts account. But we also need to approve the router to transfer funds from the contract to the makers reserve. E.g., if the contract can make offers on the DAI-USDC market, then you need to makes sure that the contract has approved Mangrove and the router for DAI and USDC transfers. [MangroveOffer](../background/offer-maker/mangrove-offer.md)'s helper functions ([activate](../technical-references/code/strategies/MangroveOffer#activate) and [checklist](../technical-references/code/strategies/MangroveOffer#checklist)) also handles router approvals.
+#### When a Forwarder-based contract manages offers belonging to several offer owners
 
-```mermaid
-sequenceDiagram
+The [Forwarder](../background/offer-maker/forwarder.md) building block provides a base for maker contracts that manages offers belonging to several offer owners.
 
-    participant Maker as Makers reserve
-    participant MC as Maker Contract
+This is an example of case, where the %%offer reserve|reserve%% of offer owners will not reside directly on the maker contract. In such cases, the offer owners need to approve the maker contract or its %%router|router%% (if it has one) for outbound token transfer. 
```

Comment:
Forwarder strat do not impose that offer owner's reserve is not the maker contract itself. I would say something like:

A basic Forwarder strategy is to assume that each offer owner's reserve is actually the offer owner's address.  In such cases [...]

Reply:
Ah, I've been thinking about Forwarder as always having offer owner reserves at another place, but I guess, that's technically not required. I guess `reserve(maker)` could be the Forwarder itself.... Hmmm. Fair, I'll go with your version.